### PR TITLE
Modularize WebUI

### LINKED ISSUE #8449: Modularize WebUI
Make the webUI a go module to allow different webserver implementations on top of the OSS

### PR DESCRIPTION
**Pull request by N-o-Z**
Closes #8449

### CODE REVIEW

File: go.work.sum [hunk 1]
```diff
@@ -0,0 +1,633 @@
+cloud.google.com/go/accessapproval v1.7.4 h1:ZvLvJ952zK8pFHINjpMBY5k7LTAp/6pBf50RDMRgBUI=
```
Comment (nadavsteindler) [inline go.work.sum:1]: How do these get updated? Is it possible that someone will bump a version but forget this file?

File: go.work [hunk 2]
```diff
@@ -0,0 +1,6 @@
+go 1.22
+
```
Comment (nadavsteindler) [inline go.work:2]: What is this file? Is it possible to add a comment at the beginning saying what it is?

**Review by nadavsteindler — APPROVED** [review]

File: go.work.sum (hunk 1 above)
Comment (N-o-Z) [reply go.work.sum:1]: go.work.sum is updated automatically in the same way go.sum is

File: go.work (hunk 2 above)
Comment (N-o-Z) [reply go.work:2]: The go.work file defines the go workspace which allows working on several modules using the local paths for them.
The go.work just like the go.mod are part of the go ecosystem - I don't think there's a need to document what a go.work file just as there's no need to document a go.mod file.
There are plenty of documentation and guides out there

**Pull request merged**